packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -940,40 +940,14 @@ mod tests {
         document().body().unwrap().append_child(&parent).unwrap();
 
         let node_ref = NodeRef::default();
-        let mut elem: VNode = html! { <div ref=node_ref.clone()></div> };
+        let mut elem: VNode = html! { <div ref={node_ref.clone()}></div> };
         assert_vtag(&mut elem);
         elem.apply(&scope, &parent, NodeRef::default(), None);
         let parent_node = parent.deref();
         assert_eq!(node_ref.get(), parent_node.first_child());
         elem.detach(&parent);
         assert!(node_ref.get().is_none());
     }
-
```

Comment:
This PR introduces a breaking change.
Should I update the code to allow this test to pass with braces inserted?

Reply:
I think breaking changes are fine. Yew makes no version guarantees right now (v0.x) and making breaking changes where needed is a good idea.